src/main/js/controls/Graph/Graph.js

```diff
@@ -274,10 +276,20 @@ class Graph extends Construct {
      *  @returns {Graph} - Graph instance
      */
     resize() {
+        let graphHasData = false;
         setCanvasWidth(this.graphContainer, this.config);
         scaleGraph(this.scale, this.config);
         translateGraph(this);
-        this.content.forEach((control) => control.resize(this));
+        removeNoDataView(this.svg);
```

Comment:
Do you need to remove on each resize? Isnt it ideal to only add/remove when there is no data/data loaded respectively?

Reply:
I have removed the code here and updated to use create and translate phase